systemtest/src/test/java/io/strimzi/systemtest/report/DefaultClusterReportST.java

```diff
@@ -0,0 +1,448 @@
+            .withReplicas(2)
+            .createInstallation()
+            .runInstallation();
+        this.testStorage = new TestStorage(extensionContext);
+        resourceManager.createResourceWithWait(extensionContext,
+            KafkaTemplates.kafkaWithCruiseControl(testStorage.getClusterName(), 3, 3).build(),
+            KafkaTemplates.kafkaEphemeral(testStorage.getClusterName() + "-tgt", 3).build()
+        );
+        resourceManager.createResourceWithWait(extensionContext,
+            KafkaTopicTemplates.topic(testStorage.getClusterName(), "my-topic", 1, 1, testStorage.getNamespaceName()).build(),
+            KafkaUserTemplates.tlsUser(testStorage.getNamespaceName(), testStorage.getClusterName(), "my-user").build(),
+            KafkaRebalanceTemplates.kafkaRebalance(testStorage.getClusterName()).build(),
+            KafkaBridgeTemplates.kafkaBridge("my-bridge", KafkaResources.plainBootstrapAddress(testStorage.getClusterName()), 1).build(),
+            KafkaConnectTemplates.kafkaConnect("my-connect", testStorage.getNamespaceName(), testStorage.getClusterName(), 1).build(),
+            KafkaMirrorMaker2Templates.kafkaMirrorMaker2("my-mm2", testStorage.getClusterName() + "-tgt", testStorage.getClusterName(), 1, false).build()
+        );
+    }
+
+    private void assertValidClusterRoleBindings(String outPath) throws IOException {
+        for (String s : Arrays.asList(
+            "strimzi-cluster-operator.yaml",
+            "strimzi-cluster-operator-kafka-broker-delegation.yaml",
+            "strimzi-cluster-operator-kafka-client-delegation.yaml"
+        )) {
+            assertValidYamls(outPath + "/reports/clusterrolebindings", ClusterRoleBinding.class, s, 1);
+        }
+    }
+
+    private void assertValidClusterRoles(String outPath) throws IOException {
+        for (String s : Arrays.asList(
+            "strimzi-cluster-operator-global.yaml",
+            "strimzi-cluster-operator-leader-election.yaml",
+            "strimzi-cluster-operator-namespaced.yaml",
+            "strimzi-cluster-operator-watched.yaml",
+            "strimzi-entity-operator.yaml",
+            "strimzi-kafka-client.yaml",
+            "strimzi-kafka-broker.yaml"
+        )) {
+            assertValidYamls(outPath + "/reports/clusterroles", ClusterRole.class, s, 1);
+        }
+    }
+
+    private void assertValidConfigMaps(String outPath, String clusterName) throws IOException {
+        for (String s : Arrays.asList(
+            "my-bridge-bridge-config.yaml",
+            clusterName + "-cruise-control-config.yaml",
+            clusterName + "-entity-topic-operator-config.yaml",
+            clusterName + "-entity-user-operator-config.yaml",
+            clusterName + "-kafka-0.yaml",
+            clusterName + "-kafka-1.yaml",
+            clusterName + "-kafka-2.yaml",
+            clusterName + "-zookeeper-config.yaml",
+            "my-connect-connect-config.yaml",
+            "my-mm2-mirrormaker2-config.yaml"
```

Comment:
do we want to check the `strimzi-cluster-operator` ConfigMap as well?

Reply:
The script doesn't get this. Do we want to add it in this PR?